AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/AWSIncomingEventReconciliationQueue.swift

```diff
@@ -66,6 +69,12 @@ final class AWSIncomingEventReconciliationQueue: IncomingEventReconciliationQueu
                 $0.modelSchema.name == modelName
             })
             let modelPredicate = syncExpression?.modelPredicate() ?? nil
+            guard reconciliationQueues[modelName] == nil else {
+                Amplify.DataStore.log
```

Comment:
I see references to both `Amplify.log` and `Amplify.DataStore.log` in this file. Is that intentional?

ps: thanks for adding more informative logging!